ipa-multipoint/ipa_multipoint_jni/src/lib.rs

```diff
@@ -145,10 +145,27 @@ pub extern "system" fn Java_org_hyperledger_besu_nativelib_ipamultipoint_LibIpaM
         .convert_byte_array(input)
         .expect("Cannot convert jbyteArray to rust array");
 
-    let committer = &CONFIG.committer;
-
-    let (old_commitment_bytes,indexes,old_scalars,new_scalars) = ffi_interface::deserialize_update_commitment_sparse(input);
-    let updated_commitment = ffi_interface::update_commitment_sparse(committer, old_commitment_bytes, indexes, old_scalars, new_scalars).unwrap();
+    let (old_commitment_bytes, indexes, old_scalars, new_scalars) =
+        match ffi_interface::deserialize_update_commitment_sparse(input) {
+            Ok(decomposed_input) => decomposed_input,
+            Err(err) => {
+                env.throw_new(
+                    "java/text/ParseException",
+                    format!("Could not deserialize the input, error : {:?}", err),
+                )
+                .expect("Failed to throw exception");
+                return std::ptr::null_mut(); // Return null pointer to indicate an error
+            }
+        };
+
+    let updated_commitment = ffi_interface::update_commitment_sparse(
+        &CONFIG,
```

Comment:
As of the recent rust-verkle -- most methods just take a `CONFIG` to make the downstream callers job easier and the API simpler